aws/rust-runtime/aws-config/src/sso/token.rs

```diff
@@ -321,14 +321,10 @@ impl Builder {
         if self.sdk_config.is_none() {
             self.sdk_config = Some(crate::load_defaults(crate::BehaviorVersion::latest()).await);
         }
-        self.build_with(Env::real(), Fs::real())
+        self.build_sync(Env::real(), Fs::real())
     }
 
-    pub(crate) fn build_sync(self) -> SsoTokenProvider {
```

Comment:
`build_sync` is a little bit a confusing name since its actual functionality is "build without loading new credentials from the environment, not "build but use spawn_blocking" or something along those lines.

Maybe `build_as_configured` or something? Could also just go back to `build_with` as the function name

Reply:
Fixed in 3ab8cfac3748f7e42e1707abe80e5425ffa0ba95